Doc/library/exceptions.rst

```diff
@@ -725,31 +725,29 @@ depending on the system error code.
 
    Raised when a file operation (such as :func:`os.remove`) is requested
    on a directory.
-   Corresponds to :c:data:`errno` ``EISDIR``.
+   Corresponds to :c:data:`errno` :py:data:`~errno.EISDIR`.
 
 .. exception:: NotADirectoryError
 
-   Raised when a directory operation (such as :func:`os.listdir`) is requested on
-   something which is not a directory.  On most POSIX platforms, it may also be
-   raised if an operation attempts to open or traverse a non-directory file as if
-   it were a directory.
-   Corresponds to :c:data:`errno` ``ENOTDIR``.
+   Raised when a directory operation (such as :func:`os.listdir`) is requested
+   on something which is not a directory.
```

Comment:
```suggestion
   Raised when a directory operation (such as :func:`os.listdir`) is requested on
   something which is not a directory.  On most POSIX platforms, it may also be
   raised if an operation attempts to open or traverse a non-directory file as if
   it were a directory.
```
Bad merge